### ISSUE TITLE
Doublons avec position identique

### ISSUE THREAD

**Issue by cquest** [etalab/ban-data#4]
Sur le premier fichier de test, il y a 25359 doublons comportant le même X/Y.

Exemple: 
33014_B035_41_C : 64 fois
33005_0682_54_ : 56 fois

**Comment by FrancoisVIGN** [comment]
Ce sont les adresses que nous ne sommes pas capable de localiser et qui se retrouvent empilées ?. A quoi correspond le code ? à la CIA ? ne manque t il pas un nom de voie ?

**Comment by cquest** [comment]
Oui, le code correspond à la CIA que j'avais commencé à regénérer sur le premier fichier vu que l'id n'était pas présent.

Ces cas portent le même numéro (41 C pour le premier exemple et 54 pour le second).
Les champs sont tous identiques, sauf l'ID... donc de purs doublons.

**Comment by FrancoisVIGN** [comment]
Il s'agit pour la plupart d'adresses internes qui n'ont pas d'intérêt en externe.

**Comment by FrancoisVIGN** [comment]
il s'agit le plus souvent de place de parking qui sont à la même adresse mais qui sont sur des parcelles différentes.

**Comment by cquest** [comment]
Et donc ?
Ajouterez vous un filtrage avant la sortie BAN ou sinon comment faire la sélection de l'adresse qui ne correspond pas à un parking (ou autre) ?

**Comment by ghost** [comment]
on a aussi parfois des lotissements qui sont empilés comme ça ou alors des résidences. On peut décider qu'un seul point sort mais au final cela correspond bien à n objets différents sur le terrain. A décider pour qu'on adapte ou non le script sur ce point-là

**Comment by cquest** [comment]
Si ces N objets ont des attributs différents c'est normal de les avoir N fois y compris à la même position, mais dans le cas où l'on a exactement les mêmes attributs dans les autres champs, il est quand même plus cohérent d'un point de vue utilisateur des données de n'avoir d'un seul objet en sortie.

**Comment by ghost** [comment]
oui mais c'est parce que nous avons d'autres attributs dans nos bases et puis la position est différente c'est aussi un attribut mais nous allons les supprimer pour que ce soit plus simple pour l'utilisateur

**Comment by ghost** [comment]
normalement on a gardé un seul élément par pile quand même numéro.

**Comment by cquest** [comment]
Le 54 Boulevard de la Plage à Andernos-les-Bains est toujours présent 56 fois dans la livraison du 10/4/2015.

id ADRNIVX_0000000276423751 à ADRNIVX_0000000276423806

**Comment by parville** [comment]
Avec pareil nom de voie, pas étonnant d'avoir un grain de sable ! :innocent:

**Comment by parville** [comment]
Dans la BAN 76 de Sept 2015 on trouve des numéros de voies en doublons avec des coordonnées différentes en secteur rural. 
Exemple : _chemin du mont calo_ commune de _le mesnil-lieubray_ .

Les doublons sont liés à une variante avec/sans un nom de lieu-dit.
Pour l'un des doublons j'ai une 3ème variante à proposer suite acquisition adresse postale via opérateur !
